src/Factory.php

```diff
@@ -32,7 +32,7 @@ class Factory extends BaseFactory
      */
     public function writer($path = TMP)
     {
-        return parent::writer($path);
+        return parent::writer($this->config->get('general.timestampPath') ?: $path);
```

Comment:
One thing I realized after merging the mini-asset change, is that this path also moves the temporary files created in development mode.